src/oge/helpers.py

```diff
@@ -325,11 +389,49 @@ def assign_fuel_category_to_ESC(
     return df
 
 
-def add_plant_local_timezone(df, year):
-    plant_tz = load_data.load_pudl_table(
-        "plants_entity_eia", columns=["plant_id_eia", "timezone"]
+def add_plant_entity(df: pd.DataFrame) -> pd.DataFrame:
+    """Adds timezone and geographical information to input data frame.
+
+    Args:
+        df (pd.DataFrame): table with a 'plant_id_eia' column.
+
+    Returns:
+        pd.DataFrame: original data frame with additional 'timezone', 'lat', 'lon',
+            'state', 'county', 'city', 'zip_code' and 'street_address' columns
+    """
+    geographical_info = [
+        "latitude",
+        "longitude",
+        "state",
+        "county",
+        "city",
+        "zip_code",
+        "street_address",
```

Comment:
I would propose removing street address and zip code, but adding the `plant_name_eia` column.
I don't think we need street address since we have lat/long info

Reply:
Sounds good

Reply:
Done

<img width="1797" alt="Screenshot 2024-05-22 at 9 46 54 AM" src="https://github.com/singularity-energy/open-grid-emissions/assets/16549571/3c8b4c0d-62a1-4611-aaf0-063a9271925f">